docs/IR.md

```diff
@@ -392,7 +392,7 @@ It is common to represent a tensor as a nested list. This generally works fine,
 
 |Group|Types|Description|
 |---|---|---|
-Floating Point Types|float16, float32, float64|Values adhering to the IEEE 754-2008 standard representation of floating-point data.
+Floating Point Types|float16, float32, float64, bfloat16, floate4m3, floate5m2|Values adhering to the IEEE 754-2008 standard representation of floating-point data or defined in paper [FP8 Formats for Deep Learning](https://arxiv.org/abs/2209.05433)
```

Comment:
Reference the GraphCore paper as well? Since the handling of spcial values will be treated as an implementation detail.

Reply:
You mean this paper: https://arxiv.org/pdf/2206.02915.pdf. The float8 number implemented are slightly different. There is no (0 and -0) as opposed to the solution chosen by NVIDIA. I was afraid that quoting two papers may confuse readers.